src/components/components/AddComponent.js

```diff
@@ -65,45 +68,38 @@ export default class AddComponent extends React.Component {
           }
         });
       });
-      var registryOptions = registryComponents
-        .sort(function (a, b) {
-          return a.componentName >= b.componentName;
-        })
-        .map(function (item) {
-          return <option key={item.componentName} origin='registry'
-            value={`${item.componentPackageName}.${item.componentName}`}>{item.componentName}</option>
+    var registryOptions = registryComponents
+      .sort(function (a, b) {
+        return a.componentName >= b.componentName;
+      })
+      .map(function (item) {
+        return {value: item.componentPackageName + '.' + item.componentName,
+          label: item.componentName, origin: 'registry'};
       });
 
-    return [commonOptions, registryOptions];
+    this.options = commonOptions.concat(registryOptions);
   }
 
   render () {
     const entity = this.props.entity;
     if (!entity) { return <div></div>; }
 
-    var options = this.renderComponentOptions();
+    this.getComponentsOptions();
     return (
-      <Collapsible>
-        <div className='collapsible-header'>
-          <span>COMPONENTS</span>
-        </div>
-        <div className='collapsible-content'>
-          <div className='row'>
-            <span className='text'>Add</span>
-            <span className='value'>
-              <select ref='select'>
-                <optgroup label="registered">
-                  {options[0]}
-                </optgroup>
-                <optgroup label="registry">
-                  {options[1]}
-                </optgroup>
-              </select>
-              <a className='button fa fa-plus-circle' onClick={this.addComponent}/>
-            </span>
-          </div>
+        <div className='add-component2'>
+          <Select
+            className="add-component"
+            ref="select"
+            autofocus
```

Comment:
I did a fix by using
```js
  componentDidUpdate (nextProps) {
    this.refs.select.focus();
  }
```
But I removed it because it prevents the scenegraph from taking the focus, so if you click on the scenegraph and try to use the arrow keys to move up and down it doesn't affect the scenegraph but the selectbox.
I've created another issue (https://github.com/aframevr/aframe-inspector/issues/342) to discuss some ideas on how to improve it, maybe detect that you've clicked the objects in the viewport and focus the add component, but if you click on the scenegraph just keep the focus there.